<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VProductPage</class>
 <widget class="QWidget" name="VProductPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1749</width>
    <height>736</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>401</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="horizontalHeaderFormat">
    <enum>QCalendarWidget::ShortDayNames</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
   <property name="dateEditEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>811</width>
     <height>721</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbDetail">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>10</y>
     <width>531</width>
     <height>721</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnYear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Display Year</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
